docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,281 @@
+
+The kubelet can optionally perform and react to two kinds of probes on running
+Containers:
+
+* `livenessProbe`: Indicates whether the Container is running. If
+   the liveness probe fails, the kubelet kills the Container, and the Container
+   is subjected to its [restart policy](#restart-policy). If a Container does not
+   provide a liveness probe, the default state is `Success`.
+
+* `readinessProbe`: Indicates whether the Container is ready to service requests.
+   If the readiness probe fails, the endpoints controller removes the Pod's IP
+   address from the endpoints of all Services that match the Pod. The default
+   state of readiness before the initial delay is `Failure`. If a Container does
+   not provide a readiness probe, the dfault state is `Success`.
+
+### When should you use liveness or readiness probes?
+
+If the process in your Container is able to crash on its own whenever it
+encounters an issue or becomes unhealthy, you do not necessarily need a liveness
+probe; the kubelet will automatically perform the correct action in accordance
+with the Pod's `restartPolicy`.
+
+If you'd like your Container to be killed and restarted if a probe fails, then
+specify a liveness probe, and specify a `restartPolicy` of Always or OnFailure.
+
+If you'd like to start sending traffic to a Pod only when a probe succeeds,
+specify a readiness probe. In this case, the readiness probe might be the same
+as the liveness probe, but the existence of the readiness probe in the spec means
+that the Pod will start without receiving any traffic and only start receiving
+traffic after the probe starts succeeding.
+
+If you want your Container to be able to take itself down for maintenance, you
+can specify a readiness probe that checks an endpoint specific to readiness that
+is different from the liveness probe.
+
+Note that if you just want to be able to drain requests when the Pod is deleted,
+you do not necessarily need a readiness probe; on deletion, the Pod automatically
+puts itself into an unready state regardless of whether the readiness probe exists.
+The Pod remains in the unready state while it waits for the Containers in the Pod
+to stop.
+
+## Pod and Container status
+
+For detailed information about Pod Container status, see
+[PodStatus](/docs/resources-reference/v1.5/#podstatus-v1)
+and
+[ContainerStatus](/docs/resources-reference/v1.5/#containerstatus-v1).
+Note that the information reported as Pod status depends on the current
+[ContainerState](/docs/resources-reference/v1.5/#containerstate-v1).
+
+## Restart policy
+
+A PodSpec has a `restartPolicy` field with possible values Always, OnFailure,
+and Never. The default value is Always.
+`restartPolicy` applies to all Containers in the Pod. `restartPolicy` only
+refers to restarts of the Containers by the kubelet on the same node. Failed
+Containers that are restarted by the kubelet are restarted with an exponential
+back-off delay. The delay is in multiples of sync-frequency (
```

Comment:
I don't know what sync-frequency refers to, but the backoff is defined as 10 seconds. (i.e. 10s, 20s, 40s, etc.)